hippunfold/workflow/rules/common.smk

```diff
@@ -67,6 +67,15 @@ def get_final_coords():
                 space=config['output_spaces'],
                 allow_missing=True)
 
+def get_final_isovol():
```

Comment:
shouldn't need this if it is already an input to create_warps..

Reply:
with the name change to use the `desc` tag, we could perhaps just use get_final_coords() to include the various methods

Reply:
fixed